include/common.h

```diff
@@ -42,6 +42,14 @@
         POS(); \
         DEBUG(__VA_ARGS__)
 
+inline int min(const int a, const int b)
```

Comment:
This should probably be `static inline` (otherwise it will break with `-O0`)

By default `inline` will create an external function call if it cannot be inlined.